src/App.vue

```diff
@@ -16,7 +16,7 @@
         </transition>
       </main>
       <es-toasts />
-      <es-footer />
+      <es-footer :current-route="this.$router.currentRoute.params.id"/>
```

Comment:
in the template, you don't have to use this. If this works, without having the correct syntax, it means that you don't need it

Reply:
Seems that this is not used anymore. I'll get rid of this.